modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -180,6 +180,24 @@ public static void setUpClass() throws Exception {
 		BaseNotificationTypeTest.setUpClass();
 	}
 
+	public List<NotificationQueueEntry> getNotificationQueueEntries() {
```

Comment:
@rijema please make it private. You don't need to expose it as public if you are not using it in other classes.